gradle.properties

```diff
@@ -2,8 +2,8 @@ dropwizardVersion=1.0.5
 # Make sure to update jersey version with dropwizard
 jerseyVersion=2.23.2
 asynchttpclientVersion=2.0.32
-zipkinAwsVersion=0.2.1
+zipkinAwsVersion=0.10.1
 awsSdkVersion=1.11.49
-stBraveVersion=0.1.14
-braveVersion=4.1.1
+stBraveVersion=0.5.1-SNAPSHOT
```

Comment:
Is there something that is not released yet?  If so we can make a release otherwise this should be 0.5.0

Reply:
just saw the PR I will try to understand why we need that more.